src/datasource/DataResourcesManager.ts

```
@@ -67,18 +67,34 @@ export class PostgresSchemaManager implements IDataLayerResourcesManager {
             const method = this.primitiveTypesMapping[gqlField.type];
             if (method) {
               table[method](gqlField.name);
-            } else {
-              // DEBT: Relationships :)
-              // DEBT: Mapping scalars
-              // DEBT: Input type support
-              logger.error(`Using unsupported field ${gqlField.name} in ${gqlType.name} type`)
             }
           })
           table.timestamps();
         })
       }
     }
 
+    for (const gqlType of types) {
+      let tableName = context.getFieldName(gqlType)
+      let currentTable = tableName
+      gqlType.fields.forEach(async(gqlField: Field) => {
+        if(gqlField.isType) {
+          if("OneToMany" in gqlField.directives) {
+            let fieldname = gqlField.directives['OneToMany'].field
+            if(gqlField.isArray) {
+              tableName = gqlField.type.toLowerCase()
+            } else {
+              currentTable = gqlField.type.toLowerCase()
```

As far I understand this I think that `OneToMany` will only apply to array cases:
```
 comment: [Comment]! @OneToMany(field: "comments_note")
```

This else for me do not make sense and we should throw an error.

---

I was thinking of another way of defining the relation, But this is better defined way.

---

Can we explain that in the description of the PR? What is the approach you are trying to take here etc.

---

If the user enters like you mentioned above:
```
type Car {
  persons: [Person!]! @OneToMany(field: "car_persons")
}
```
or if enters it like
```
type Person {
  car: Car! @OneToMany(field: "car_persons")
}
```
the else handles this case